src/Storefront/Page/Checkout/Confirm/AdyenCheckoutConfirmPageLoader.php

```diff
@@ -128,12 +128,19 @@ private function filterShopwarePaymentMethods($originalPaymentMethods, SalesChan
             //If this is an Adyen PM installed it will only be enabled if it's present in the /paymentMethods response
             if (strpos($paymentMethodEntity->getFormattedHandlerIdentifier(), 'adyen')) {
                 $pmCode = 'scheme'; //TODO get from payment method handler instead of hardcoding PM type
+                // In case the paymentMethods response has no payment methods, remove it from the list
+                if (empty($adyenPaymentMethods)) {
+                    $originalPaymentMethods->remove($paymentMethodEntity->getId());
+                    continue;
+                }
```

Comment:
Wouldn't this remove all payment methods from the list including Shopware defaults and other PMs installed?